From e57a62138c4d3e7c8ca1848a5fbc509c55242f6a Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Wed, 23 Mar 2016 15:31:36 +0200
Subject: [PATCH 45/66] defconfig: use cmdline console configuration for getty

Buildroot was configured to use the ttyS0 as the port for getty,
this created a problem for boards using the Marvell UART driver
where the device it ttyMV0.

To fix this BR2 can be configured to use default settings which
is to use the kernel cmdline (set by u-boot) console= parameter.
This way if boards uses the ttyX0, this parameter is passed to
kernel and is used by /etc/inittab getty respawn.

Change-Id: I682258e2cb17d8e860aba23334a8675365e3f1f8
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28477
Tested-by: Star_Automation <star@marvell.com>
---
 configs/mvebu_armv7_tiny_le_defconfig | 1 -
 configs/mvebu_armv8_be_defconfig      | 1 -
 configs/mvebu_armv8_le_defconfig      | 1 -
 configs/mvebu_armv8_tiny_le_defconfig | 1 -
 4 files changed, 4 deletions(-)

diff --git a/configs/mvebu_armv7_tiny_le_defconfig b/configs/mvebu_armv7_tiny_le_defconfig
index 74543ef..ece37b1 100644
--- a/configs/mvebu_armv7_tiny_le_defconfig
+++ b/configs/mvebu_armv7_tiny_le_defconfig
@@ -10,7 +10,6 @@ BR2_TOOLCHAIN_EXTERNAL_GCC_4_6=y
 BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_2=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
 BR2_MVEBU_RELEASE_NAME="2015_T3.0.eng_drop_v1"
-BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files"
 BR2_TARGET_ROOTFS_CPIO=y
 BR2_TARGET_ROOTFS_EXT2=y
diff --git a/configs/mvebu_armv8_be_defconfig b/configs/mvebu_armv8_be_defconfig
index 98172ac..e07bff1 100644
--- a/configs/mvebu_armv8_be_defconfig
+++ b/configs/mvebu_armv8_be_defconfig
@@ -7,7 +7,6 @@ BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_18=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
 BR2_TOOLCHAIN_EXTERNAL_CXX=y
 BR2_MVEBU_RELEASE_NAME="16.02.0 ED"
-BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files/"
 BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
 BR2_PACKAGE_BZIP2=y
diff --git a/configs/mvebu_armv8_le_defconfig b/configs/mvebu_armv8_le_defconfig
index 128fccf..f16ab24 100644
--- a/configs/mvebu_armv8_le_defconfig
+++ b/configs/mvebu_armv8_le_defconfig
@@ -9,7 +9,6 @@ BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_18=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
 BR2_TOOLCHAIN_EXTERNAL_CXX=y
 BR2_MVEBU_RELEASE_NAME="16.02.0 ED"
-BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files/"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_VERSION=y
diff --git a/configs/mvebu_armv8_tiny_le_defconfig b/configs/mvebu_armv8_tiny_le_defconfig
index d7cd142..615cd7f 100644
--- a/configs/mvebu_armv8_tiny_le_defconfig
+++ b/configs/mvebu_armv8_tiny_le_defconfig
@@ -7,7 +7,6 @@ BR2_TOOLCHAIN_EXTERNAL_GCC_5=y
 BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_18=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
 BR2_MVEBU_RELEASE_NAME="16.02.0 ED"
-BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files"
 BR2_ROOTFS_POST_IMAGE_SCRIPT="board/marvell/scripts/filesystem-post-script.sh"
 BR2_PACKAGE_MTD=y
-- 
1.9.1

